tests/wpt/metadata/touch-events/historical.html.ini

```diff
@@ -0,0 +1,16 @@
+[historical.html]
```

Comment:
Legit failures, `document.createTouch` disappeared in favour of a `Touch` constructor.